azure-pipelines.yml

```diff
@@ -66,6 +66,9 @@ jobs:
         - script: 'pip install sklearn-contrib-lightning'
           displayName: 'Install lightning'
 
+        - script: 'pip install --force-reinstall --no-cache-dir shap'
```

Comment:
Installing shap from release, also installs the latest scikit-learn, which is 0.24. So then building the docs crashes because it cannot run dml.py, since it dml.py imports the subsampledhonestforest which uses the deprecated scikit-learn.ensemble.forest module! 

We should just remove this installation instruction.